packages/flutter/lib/src/material/text_field.dart

```diff
@@ -543,6 +543,21 @@ class _TextFieldState extends State<TextField> with AutomaticKeepAliveClientMixi
     _startSplash(details);
   }
 
+  void _handleForcePressStarted(ForcePressDetails details) {
+    if (widget.selectionEnabled) {
+      switch (Theme.of(context).platform) {
+        case TargetPlatform.iOS:
+          // The cause is not technically double tap, but we would like to show
+          // the toolbar.
+          _renderEditable.selectWordsInRange(from: details.globalPosition, cause: SelectionChangedCause.doubleTap);
```

Comment:
why are we saying SelectionChangedCause is something other than what it is?
Shouldn't we introduce a new enum value?

cc @xster